docs/contributing.rst

```diff
@@ -148,3 +148,12 @@ Preparing Pull Requests
 
    The merged pull request will undergo the same testing that your local branch
    had to pass when pushing.
+
+#. After your pull request is merged into the `GliderTools/master`, you will need
+   to fetch those changes and rebase your master so that your master reflects the latest
+   version of GliderTools.
+
+    $ git checkout master
+    $ git fetch upstream
```

Comment:
Does the guide include instructions on how to set up the `upstream` remote?

Reply:
Also could you add some explanations what is happening in each step:
git checkout master #switching back to the main/master branch
git fetch upstream #Downloading all the changes from the central repo
git rebase upstream/master #applying the changes that were made to the original repo's master branch onto your local branch, so they match exactly.

Reply:
Yes, setting up of upstream master are explained in step 2.